client-src/default/index.js

```diff
@@ -198,9 +198,13 @@ const onSocketMsg = {
 let hostname = urlParts.hostname;
 let protocol = urlParts.protocol;
 let port = urlParts.port;
+let sockPort;
 
+if (urlParts.path !== null && urlParts.path !== '/') {
+  sockPort = querystring.parse(urlParts.path).sockPort;
+}
 // check ipv4 and ipv6 `all hostname`
-if (hostname === '0.0.0.0' || hostname === '::') {
+if ((hostname === '0.0.0.0' || hostname === '::') && !sockPort) {
```

Comment:
Why? We need test on this

Reply:
oh~~~,i misunderstand his idea
![image](https://user-images.githubusercontent.com/17424434/56081978-f6548780-5e45-11e9-9af7-4667c866a952.png)

Reply:
Please revert and keep this PR as is. Other fixes/features will be done in other PRs

Reply:
> Please revert and keep this PR as is. Other fixes/features will be do in other PRs

already revert

Reply:
:+1: